test/environments/examples/bit_flipping_env.jl

```diff
@@ -0,0 +1,8 @@
+@testset "bit_flipping_env" begin
+
+    env = BitFlippingEnv(; N = 7)
```

Comment:
Use an independent `rng` like https://github.com/JuliaReinforcementLearning/ReinforcementLearningEnvironments.jl/blob/ed9e04c7032a57f1f5961f4a608f9ec3d4d5a60c/test/environments/examples/tiger_problem_env.jl#L3-L5 here to avoid `GLOBAL_RNG` being polluted.